src/main/java/com/hubspot/jinjava/LegacyOverrides.java

```diff
@@ -9,11 +9,13 @@
   private final boolean evaluateMapKeys;
   private final boolean iterateOverMapKeys;
   private final boolean usePyishObjectMapper;
+  private final boolean useWhitespaceAfterStartToken;
```

Comment:
I think a better name might be `whitespaceRequiredWithinTokens` as it makes it sound more like it changes how input whitespace is used rather than how whitespace is output